indy_node/server/config_req_handler.py

```diff
@@ -55,6 +57,20 @@ def _doStaticValidationPoolUpgrade(self, identifier, reqId, operation):
 
         # TODO: Check if cancel is submitted before start
 
+    def curr_pkt_info(self, pkg_name):
+        if pkg_name == "indy-node":
```

Comment:
Do we have a test when package name is not equal to indy-node? Can we have all existing tests parametrized so that we test both indy-node and a custom package?

Reply:
we have some